modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -86,6 +86,7 @@ ParallelCoordinates::ParallelCoordinates()
                            vec4(.5f, .5f, .5f, 1))
     , tf_("tf", "Line Color")
     , tfSelection_("tfSelection", "Selection Color")
+    , enableHoverColor_("enableHoverColor", "Enable Hover Color", false)
```

Comment:
On by default?
How is the hover color determined?

Reply:
Sure.

The hover color is the same as the selected color:
> Note that even though hover and selected now have the same color the selected has less color falloff or wider linewidth or something that makes them look different from each other.